### DISCUSSION TITLE
OAuth version used

### DISCUSSION THREAD

**Discussion by colinbridle** [MscrmTools/XrmToolBox#1012] [Q&A]
Hi,

Just wanted to check if the version of OAuth in use by XRM ToolBox is version 2 - I have been asked to confirm this by my security colleagues.

Thanks
Colin

**Comment by MscrmTools** [comment]
I can't tell... it uses what Microsoft assemblies use

**Comment by rajyraman** [comment]
> Hi,
> Just wanted to check if the version of OAuth in use by XRM ToolBox is version 2 - I have been asked to confirm this by my security colleagues.
> Thanks
> Colin

@colinbridle - It is OAuth2. https://docs.microsoft.com/en-us/powerapps/developer/data-platform/authenticate-oauth
